README.md

```diff
@@ -168,41 +166,7 @@ return this.infinityModel("product", { perPage: 12, startingPage: 1,
                                        category: "furniture" });
 ```
 
-Moreover, you can optionally pass in an object of bound parameters as a third option to `infinityModel` to further
-customize the request to the backend. The values of the contained parameters will be looked up against the route
-properties and the respective values will be included in the request:
-
-```js
-import Ember from 'ember';
-import InfinityRoute from 'ember-infinity/mixins/route';
-
-export default Ember.Route.extend(InfinityRoute, {
-  ...
-
-  prod: Ember.computed('cat', function () { return this.get('cat'); }),
-  country: '',
-  cat: 'shipped',
-
-  model() {
-    return this.infinityModel("product", { perPage: 12, startingPage: 1, make: "original" }, { country: "country", category: "prod" });
-  }
-});
-```
-
-In the example above, the query url should look like this:
-
-```
-  product?make=original&country=&category=shipped&per_page=12&page=1
-```
-
-If the value of the bound parameter cannot be found, the parameter is not included in the request. Note that you cannot have
-a static and bound parameter of the same name, the latter will take precedence.
-
-When you need to pass in bound parameters but no static parameters or custom pagination, call `infinityModel` with an empty object for it's second param:
-
-```js
-  return this.infinityModel("product", {}, { country: "country", category: "prod" });
-```
+Bound Parameters has been DEPRECATED as the third argument to `infinityModel`.  
```

Comment:
@hhff so should we add `mergeParams` hash to the [buildParams](https://github.com/hhff/ember-infinity/blob/modularize-infinity-model/addon/lib/infinity-model.js#L106) function? 

Another option you mentioned was an official mixin that backports bound params.

What are your thoughts here?